apps/docs/src/components/AIPanel/AIPanel.tsx

```
@@ -0,0 +1,615 @@
+                    onClick={() => {
+                      navigator.clipboard.writeText(result);
+                    }}
+                    title="Copy to clipboard"
+                  >
+                    📋
+                  </button>
+                </div>
+                <div
+                  className="ai-result-content"
+                  dangerouslySetInnerHTML={{
+                    __html: formatMarkdown(result),
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)